Libraries/DirectMLX.h

```diff
@@ -476,7 +476,8 @@ namespace dml
             uint64_t totalTensorSizeInBytesVal,
             uint32_t guaranteedBaseOffsetAlignmentVal)
         {
-            assert(!tensorStrides || tensorStrides->size() == static_cast<uint32_t>(tensorSizes.size()));
+            const uint32_t dimensionCount = static_cast<uint32_t>(tensorSizes.size());
```

Comment:
This isn't right - this is undoing a change to fix Clang builds. See #78

Reply:
(ditto the other unrelated changes in this file - these are bugfixes you don't want to undo)